modules/rules-engine/example_rules1.yaml

```diff
@@ -0,0 +1,7 @@
+- name: Read sensitive file from untrusted process
+  typ: FileOpened
```

Comment:
Can we rename this to `type` while keeping the Rust code as `typ` to avoid the reserved keyword?

Reply:
sure, I'll try